DesignSystem/Sources/DesignSystem/BrandingManager+Extensions.swift

```diff
@@ -75,6 +75,9 @@ public extension Button.Style {
     static let attention = Button.Style("attention")
     static let text      = Button.Style("text")
     static let toggle    = Button.Style("toggle")
+
+    static let primaryDisabled   = Button.Style("primaryDisabled")
```

Comment:
So buttons should have a disabled state, you shouldn't be restyling for disabled. Example:
Look at `extension DefaultBrand: ButtonBrand` and I'm setting for state... disabled is a state on the level of highlighted and selected. So these buttons should you want should have 
`button.setBackgroundColor(someColor, for: .disabled)` then you can set a button disabled and it will style in this way. That's the UIKit paradigm.